### DISCUSSION TITLE
Will full thanos support be implemented for this chart?

### DISCUSSION THREAD

**Discussion by keramblock** [prometheus-operator/prometheus-operator#6745] [General]
Hi, thank you for your chart, I am using kube-prom-stack chart with prometheus-operator chart as a dependency and I want to ask if there any plans to add native thanos support for all thanos services(right now its only 2 out of 6 AFAIK) to this chart?

**Comment by keramblock** [comment]
Raised similar question in kube-prometheus-stack chart:
https://github.com/prometheus-community/helm-charts/issues/4706

**Reply by nicolastakashi** [reply]
Hey @Keramblock the kube-prometheus-stack is a different project not maintained by us, probably they might provide better insights about it.
